File: apps/mobile/app/src/main/java/com/example/cibercheck/dto/session/StudentDailyCourseDto.kt
package com.example.cibercheck.dto.session

import com.google.gson.annotations.SerializedName

data class StudentDailyCourseDto(

    @SerializedName("courseName")
    val courseName: String,

    @SerializedName("sectionName")
    val sectionName: String,

    @SerializedName("startTime")
    val startTime: String?,

    @SerializedName("endTime")
    val endTime: String?,

    @SerializedName("topic")
    val topic: String?
)

File: apps/api/Features/Sessions/Dtos/StudentDailyCourseDto.cs
namespace CiberCheck.Features.Sessions.Dtos
{
    public class StudentDailyCourseDto
    {
        public string CourseName { get; set; } = string.Empty;
        public string SectionName { get; set; } = string.Empty;
        public TimeOnly? StartTime { get; set; }
        public TimeOnly? EndTime { get; set; }
        public string? Topic { get; set; }
    }
}
